### DISCUSSION TITLE
图表在移动端页面下滑不动, 并且报错

### DISCUSSION THREAD

**Discussion by xyr550** [antvis/G2#5540] [Q&A]
![5258bef0a3c2e4aac62dec2f42f9ad0](https://github.com/antvis/G2/assets/73283715/ff2966fa-4281-439c-bdaa-10c1b1ec05b2)

**Comment by pearmini** [comment]
是多少版本的？

**Reply by xyr550** [reply]
去掉了canvans上 touch-action: none 的css设置， 移动端也可以上下滑动了
但是报错仍未消失
5.0.18版本
饼图上下滑动时出现，其他图表也有报错，但是饼图必现
